src/main/java/org/openhim/mediator/dsub/service/DsubServiceImpl.java

```diff
@@ -78,9 +81,23 @@ public void newDocumentForPullPoint(String docId, String locationId) {
         pullPoint.registerDocument(docId);
     }
 
-    @Override
+    
     public List<String> getDocumentsForPullPoint(String locationId) {
         PullPoint pullPoint = pullPointFactory.get(locationId);
         return pullPoint.getDocumentIds();
     }
+
+
+    @Override
+    public void newDocumentForPullPoint(CreatePullPoint createPullPointRequest) {
```

Comment:
Introduced these changes for this file ie (DsubServiceImpl.java)  so as to atleast have mvn clean install -DskipTests passing for a mean time cc @pmanko